common.mk

```diff
@@ -1272,7 +1272,7 @@ OPTS =
 # See benchmark/README.md for details.
 benchmark: miniruby$(EXEEXT) update-benchmark-driver PHONY
 	$(BASERUBY) -rrubygems -I$(srcdir)/benchmark/lib $(srcdir)/benchmark/benchmark-driver/exe/benchmark-driver \
-	            --executables="compare-ruby::$(COMPARE_RUBY) -I$(EXTOUT)/common --disable-gem" \
+	            --executables="compare-ruby::$(COMPARE_RUBY) -I$(EXTOUT)/common -I$(EXTOUT)/$(arch) -Ilib --disable-gem" \
```

Comment:
`COMPARE_RUBY` may not be compatible with the built extensions, and your benchmark uses no extensions.
Why did you think it was needed?

Reply:
RIght, that's what I feared, it also load the built extensions with the compare Ruby which isn't what I want.

The reason I need this is because without native extensions, I don't have access to `Encoding::UTF_8` etc.

Reply:
Nevermind, I'm stupid. I had this problem because I was using a standalone `ruby` executable to compare, it works fine if I use a fully installed ruby instead.

I'll remove this part.